### ISSUE TITLE
Failed to list WordpressList.items

### ISSUE THREAD

**Issue by clanstyles** [presslabs/wordpress-operator#16]
```
E0315 14:48:30.998053       1 reflector.go:134] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:126: Failed to list *v1alpha1.Wordpress: v1alpha1.WordpressList.Items: []v1alpha1.Wordpress: v1alpha1.Wordpress.Spec: v1alpha1.WordpressSpec.Env: []v1.EnvVar: v1.EnvVar.ValueFrom: v1.EnvVarSource.SecretKeyRef: readObjectStart: expect { or n, but found ", error found in #10 byte of ...|tKeyRef":"mysite-mys|..., bigger context ...|RD","valueFrom":{"key":"PASSWORD","secretKeyRef":"mysite-mysql"}}],"envFrom":[],"ingressAnnotations"|...
E0315 14:48:30.998053       1 reflector.go:134] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:126: Failed to list *v1alpha1.Wordpress: v1alpha1.WordpressList.Items: []v1alpha1.Wordpress: v1alpha1.Wordpress.Spec: v1alpha1.WordpressSpec.Env: []v1.EnvVar: v1.EnvVar.ValueFrom: v1.EnvVarSource.SecretKeyRef: readObjectStart: expect { or n, but found ", error found in #10 byte of ...|tKeyRef":"mysite-mys|..., bigger context ...|RD","valueFrom":{"key":"PASSWORD","secretKeyRef":"mysite-mysql"}}],"envFrom":[],"ingressAnnotations"|...
```

I've double checked. RBAC is enabled. I've installed it via Helm on Minikube and am receiving this error. I've also tried giving the ServiceAccount cluster-admin just to see if that resolves it.

**Comment by Berndinox** [comment]
+1 Same here on GKE

**Comment by calind** [comment]
Fixed in 4c0094828df72279df606a139b0d8ad9a8393484.
